drools-parser/src/test/java/org/drools/parser/DRLExprParserTest.java

```diff
@@ -38,13 +38,12 @@ public class DRLExprParserTest {
 
     DrlExprParser parser;
 
-    @Before
+    @BeforeEach
     public void setUp() throws Exception {
-        new EvaluatorRegistry();
```

Comment:
Not sure what this is for. It was also removed from the `MiscDRLParserTest` so I removed it here as well. In the original `RuleParserTest` there is a comment that says "initializes pluggable operators".

Reply:
I think `EvaluatorRegistry` is not required in parser phase, it's fine to remove.